actions/extractor/codeql-extractor.yml

```diff
@@ -8,7 +8,8 @@ build_modes:
   - none
 file_coverage_languages: []
 github_api_languages: []
-scc_languages: []
+scc_languages:
+  - YAML
```

Comment:
I can see that this is now consistent with other languages. What effect does adding this have?

Reply:
This tells scc to pick up files marked as YAML when CodeQL scans for the baseline set of files.  These are then further filtered down by the configure baseline script.